src/main/java/controller/AuthorController.java

```diff
@@ -0,0 +1,19 @@
+package controller;
+
+import model.Author;
+import service.AuthorDataService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class AuthorController {
+
+    @Autowired
+    private AuthorDataService authorDataService;
+
+    @GetMapping("/authors/{authorID}")
```

Comment:
should use @RequestMapping("/authors")

cause in case of more then 1 endpoints you will duplicate the 'authors' for each of them